packages/venia-concept/src/actions/app.js

```diff
@@ -1,15 +1,30 @@
-import timeout from 'src/util/timeout';
-import {
-    drawerClose as closeMs,
-    drawerOpen as openMs
-} from 'src/shared/durations';
-
-export const toggleDrawer = drawerName => dispatch => {
-    dispatch({
-        type: 'TOGGLE_DRAWER',
-        payload: drawerName
-    });
-    return timeout(drawerName ? openMs : closeMs);
-};
-
-export const closeDrawer = () => toggleDrawer(null);
+import { createActions } from 'redux-actions';
+
+import { store } from 'src';
+
+const prefix = 'APP';
+const actionTypes = ['TOGGLE_DRAWER'];
+
+const actions = createActions(...actionTypes, { prefix });
+export default actions;
```

Comment:
This is a standard way to create both action types (string constants that reducers look for) and their corresponding boilerplate action creators. Almost all sync action creators have a sort of identity signature:

- It returns an action object
- It defines a a `payload` prop on that object, whose value is the first argument
- If `payload` is an `Error`, it defines an `error` prop, whose value is `true`

[The docs suggest](https://redux.js.org/recipes/reducingboilerplate) that exporting action types and action creators can reduce boilerplate and keep action logic centralized. Here, the action creators generated by `createActions` are named such that they can act as action types.

```js
actions.toggleDrawer.toString(); // => 'APP/TOGGLE_DRAWER'
```